generators/entity/templates/client/angular/src/main/webapp/app/entities/_entity-management.state.ts

```diff
@@ -5,7 +5,6 @@ for (var idx in fields) {
         i18nToLoad.push(fields[idx].enumInstance);
     }
 }
-i18nToLoad.push('global');
```

Comment:
I think global will be required as we use some of the keys from it. You will notice them missing if you reload I guess.